### PR TITLE
Added rsync as final step in the built Dockerfile

### PR DESCRIPTION
**Pull request by oudeismetis**

### CODE REVIEW

**Comment by oudeismetis** [comment]
still testing, cleaning up, and need to resolve conflicts.

File: build-api/api.py [hunk 1]
```diff
@@ -68,12 +66,15 @@ def build():
                 raise
 
         # Create a project specific Dockerfile from our template
+        #TODO - create .env for username@remote_host:destination_directory
+        remote_location = "/"
```
Comment (JulianGindi) [inline build-api/api.py]: This should be an env var

File: build-api/api.py [hunk 2]
```diff
@@ -28,7 +26,7 @@ def build_docker_container():
         shell=True
     )
     stopscript.wait()
-    shutil.rmtree(tmp_dir)
+    shutil.rmtree('tmp/')
```
Comment (oudeismetis) [inline build-api/api.py:30]: This should reuse the same var we set before

**Comment by JulianGindi** [comment]
:+1:

**Pull request closed without merging**